challenger/lnd.go

```diff
@@ -263,7 +239,9 @@ func (l *LndChallenger) Stop() {
 // request (invoice) and the corresponding payment hash.
 //
 // NOTE: This is part of the mint.Challenger interface.
-func (l *LndChallenger) NewChallenge(price int64) (string, lntypes.Hash, error) {
+func (l *LndChallenger) NewChallenge(price int64) (string, lntypes.Hash,
```

Comment:
Re the above comment, we could pass in the additional context here as an optional argument. Then the invoice creation has a chance to tag any extra details.